core/tests/protractor/richTextComponents.js

```diff
@@ -55,18 +55,18 @@ describe('rich-text components', function() {
     player.expectContentToMatch(function(richTextChecker) {
       richTextChecker.readBoldText('bold');
       richTextChecker.readPlainText(' ');
-      richTextChecker.readRteComponent('Link', 'http://google.com/', true);
-      richTextChecker.readRteComponent('Math', 'abc');
-      richTextChecker.readRteComponent('Video', 'ANeHmk22a6Q', 10, 100, false);
-      richTextChecker.readRteComponent(
-        'Collapsible', 'title', forms.toRichText('inner'));
       richTextChecker.readRteComponent('Tabs', [{
         title: 'title 1',
         content: forms.toRichText('contents 1')
       }, {
         title: 'title 1',
         content: forms.toRichText('contents 2')
       }]);
+      richTextChecker.readRteComponent(
```

Comment:
I could not get the tests to pass with the proper arrangement of reading, so I reversed them and the tests corrected on my local machine (But not on Travis). I will change back and also see if I change the innerHtml to what you suggested below might correct my local problem. Even if it does not work on my machine, as long as it works in Travis I'm happy.